Sources/Sentry/SentryAppStateManager.m

```diff
@@ -67,7 +72,7 @@ - (void)storeCurrentAppState
     [self.fileManager storeAppState:[self buildCurrentAppState]];
 }
 
-- (void)removeCurrentAppState
+- (void)removeAppState
```

Comment:
Since this is getting renamed, can we change it to use `delete` along with the other methods?
```suggestion
- (void)deleteAppState
```

Reply:
Sure thing